worlds/meritous/Rules.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022 FelicitusNeko
+#
+# This software is released under the MIT License.
+# https://opensource.org/licenses/MIT
+
+import typing
+from ..generic.Rules import add_rule
+from .Regions import meritous_regions, connect_regions
+
+
+def check_endgame(state, player):
+    for check in ["Meridian", "Ataraxia", "Merodach"]:
+        if not state.can_reach(check, "Location", player):
+            return False
+    return True
+
+
+def set_rules(world, player):
```

Comment:
There is no connection into the PSI Keys region, so it would never be accessible. (Though I recommend scrapping these regions entirely; see comment in Regions.py.)